internal/controller/nfspvc_controller.go

```diff
@@ -18,13 +18,18 @@ package controller
 
 import (
 	"context"
+	goerrors "errors"
```

Comment:
I'd rather you keep this package using the same name and rename the `apimachinery` errors package as `apierrors`